### ISSUE TITLE
Handle multi-organization

### ISSUE THREAD

**Issue by Max-7** [Scille/parsec-cloud#5626]
Be able to log in into multiple organization at the same time. Be able to switch between orgs.

**Comment by Max-7** [comment]
No idea how to handle this. Since the app is stateless (it gets its information from the URL and calls the bindings), my first idea was to store the whole router state, and create a new one for the new org. Something like:

```typescript
let orgStates = [];

function switchToLoginPage() {
    orgStates.push({clientHandle: router.clientHandle, history: router.history, currentRoute: router.currentRoute});
    router.clearHistory();
    router.replace('home');
}

function getLoggedInOrgs() {
    return orgStates.map(async (state) => state.clientHandle);
}

function switchToOrg(clientHandle) {
    const state = orgStates.find((state) => state.clientHandle === clientHandle);

    if (state) {
        orgStates.push({clientHandle: router.clientHandle, history: router.history, currentRoute: router.currentRoute});

        orgStates.remove(state);
        router.history = state.history;
        router.clientHandle = state.clientHandle;
        router.replace(currentRoute);
    }
}
```
But this doesn't seem possible with vue-router. The history cannot be overwritten or cleared, which is a big problem because we don't want to be able to navigate between orgs with the history.

**Comment by Max-7** [comment]
Create the branch https://github.com/Scille/parsec-cloud/tree/ms-multi-org, but what's on it doesn't work.

The idea was to use multiple apps and mount/unmount them depending on the state. Use a `LoginApp` for the main page, and then a `LoggedInApp` for each connected device, and mount/unmount each app when logging in/logging out or switching org. Each app has its own router, meaning the history between them should be preserved.
Sadly, Vue does not handle unmounting well, or rather, you can't re-mount an app you unmounted, you have to create a new one from scratch, which is frankly useless.

I then tried by mounting each app once on its own element in ths DOM and switch the element visibility depending on which app is active. But this also fails, ionic elements like popovers or modals don't seem to like it at all.

tl;dr; I've tried a few things to get the multi-org to work but it doesn't and I'm all out of ideas.
